arches/app/utils/permission_backend.py

```diff
@@ -142,6 +142,9 @@ def get_permission_search_filter(self, user): ...
     @abstractmethod
     def get_search_ui_permissions(self, user, search_result, groups=None): ...
 
+    @abstractmethod
```

Comment:
Hey Galen. I haven't looked too closely yet, but can you share why would we need a separate method if the signature is the same as `get_search_ui_permissions`?

Reply:
I actually ended up just updating the original logic of the `get_search_ui_permissions` method to handle either a `dict` or a `list` of search results and return back the like.